readchar/_macos_key.py

```diff
@@ -0,0 +1,4 @@
+# common
+
```

Comment:
please import exsiting definitions from other files here
```suggestion
from ._posix_key import *
```